src/components/WishlistPage/WishlistItem/WishlistItem.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import cx from 'classnames';
+
+import './WishlistItem.scss';
+
+const CN = 'wishlist-item';
+
+const WishlistItem = ({ item, removeFromWishlist }) => {
+  const { photo, title, collection, category } = item;
+
+  return (
+    <div className={`${cx(CN)}`}>
+      <img className={`${CN}__image`} src={photo} alt={title}></img>
+      <h3 className={`${cx(CN)}__title`}>{title}</h3>
+      <h4 className={`${cx(CN)}__subtitle`}>{collection}</h4>
+      <h4 className={`${cx(CN)}__subtitle`}>{category}</h4>
```

Comment:
You used `cx()` function a lot, but why do you need it? Template literal is enough in these cases

Reply:
`cx()` should be used for conditional classes

Reply:
Sorry! Will correct it straight away)